instance/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "TRANSFORMATIONS_DATABASE_URI": "mongodb://mongodb:27017",
+  "TRANSFORMATIONS_DATABASE_NAME": "transformations_catalog_test",
```

Comment:
Maybe shorten this to catalog or transformations_catalog (same as image name). No need for _test.

Reply:
I chose this specifically to not clash with any other databases that might be in use. The name is long, but an easy change (catalog_test?). Though as a sample stripped down config file, maybe using the suggested name makes sense.

Reply:
problem is that people might just use this as the default.

Reply:
True. I wanted to include a default config.json so that testing is as easy as possible, but that does mean that it will be used as a default for everyone. I like having the test appended for identification of changing the defaults, but it's not necessary.